wdl/wdl_options.proto

```diff
@@ -32,6 +32,8 @@ import "google/protobuf/descriptor.proto";
 import "google/protobuf/any.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
+import public "storage/datapol/annotations/proto/semantic_annotations.proto";
```

Comment:
As far as I remember when @mrjerryjohns scrubbed OpenWeave WDLC, these two Protobuf headers are Google-internal-only and are not open source and, consequently, are not appropriate to introduce as external dependencies on the public open source project.

Has that since changed?

Reply:
You remember correctly. I talked with Jerry and was told there's a public version available but I am struggling to find it.
And I can't get this CL to compile yet.
@mrjerryjohns FYI.